### PR TITLE
docs: document DevTools extensions API

### PR DESCRIPTION
**Pull request by OrKoN**
Merge after https://chromium-review.googlesource.com/c/chromium/src/+/3695394/

### CODE REVIEW

File: site/en/docs/extensions/reference/devtools_recorder/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+api: devtools.recorder
+title: chrome.devtools.recorder
+---
+
+See [DevTools APIs summary][1] for general introduction to using Developer Tools APIs.
+
+## Overview
+
+`devtools.recorder` API is a preview feature that allows extending the Recorder panel in Chrome DevTools.
```
Comment (jecfish) [inline site/en/docs/extensions/reference/devtools_recorder/index.md:10]:
```suggestion
`devtools.recorder` API is a preview feature that allows extending the [Recorder panel](/docs/devtools/recorder/) in Chrome DevTools.
```

**Review by jecfish — APPROVED** [review]

**Comment by Hilel-Bernstein** [comment]
plewase open youtube

File: site/en/docs/extensions/reference/devtools_recorder/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## Overview
+
+`devtools.recorder` API is a preview feature that allows you to extend the [Recorder panel](/docs/devtools/recorder/) in Chrome DevTools.
+Currently, you can extend only the export feature.
+
+To register an extension plugin, use the `registerRecorderExtensionPlugin` function.
+This function requires a plugin instance that implements the following:
+
+- `stringify` and `stringifyStep` functions
+- `name` and `mediaType` parameters
```
Comment (OrKoN) [inline site/en/docs/extensions/reference/devtools_recorder/index.md:17]: That is not technically correct now as the plugin instance does not implement those. Can we rephrase it a bit more?

Comment (OrKoN) [inline site/en/docs/extensions/reference/devtools_recorder/index.md:17]:
```suggestion
To register an extension plugin, use the `registerRecorderExtensionPlugin` function. This function requires a plugin instance, a `name` and a `mediaType` as parameters. The plugin instance must implement two methods: `stringify` and `stringifyStep`.
```

Comment (sofiayem) [reply site/en/docs/extensions/reference/devtools_recorder/index.md]: I beg your pardon. Apparently, I didn't get that.

**Comment by OrKoN** [comment]
The auto-generated data should be available now (once the preview regenerates). PTAL again.

**Review by sofiayem — APPROVED** [review]

**Pull request merged**